### PR TITLE
Sendgrid event

### PR DESCRIPTION
**Pull request by martinraison**
@eishay @leogrim

### CODE REVIEW

**Comment by eishay** [comment]
Congrats for the first commit!!!

**Comment by andrewconner** [comment]
Nice job!

image

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/SendgridCommander.scala [hunk 1]
```diff
@@ -32,6 +38,24 @@ class SendgridCommander @Inject() (
   }
 
   private def report(event: SendgridEvent): Unit = {
-    // todo(Martin): send a user event to Heimdal of type WAS_NOTIFIED, action event.event, channel email cf. MessagingAnalytics.clearedNotification
+    event.event foreach { eventType =>
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/SendgridCommander.scala:41]: maybe use for comprehension here? instead of 3 nested foreach?

**Comment by raymondng** [comment]
Awesome job!

**Comment by martinraison** [comment]
Thanks! @yingjieMiao: you're right - I have another commit coming soon that will fix this

**Comment by leogrim** [comment]
For comprehensions ruuuuuuule ;)
@stkem

**Comment by stkem** [comment]
:-P

**Pull request merged**